docs/contributing-to-airbyte/building-new-connector/README.md

```diff
@@ -1,47 +1,54 @@
 # Building New Connectors
 
-A connector takes the form of a Docker image which follows the [Airbyte specification](../../architecture/airbyte-specification.md).
+Airbyte supports 2 types of connectors: Sources and Destinations. A connector takes the form of a Docker image which follows the [Airbyte specification](../../architecture/airbyte-specification.md).
 
-We support 2 types of connectors: Sources and Destinations.  
-To build a new connector, we provide templates so you don't need to start everything from scratch.
+To build a new connector in Java or Python, we provide templates so you don't need to start everything from scratch. 
 
-**Please note that, at no point in time, we will ask you to maintain your connector.** The goal is that the Airbyte team and the community helps maintain the connector.
+**Note: you are not required to maintain the connectors you create.** The goal is that the Airbyte core team and the community help maintain the connector.
 
 ## The Airbyte specification
 
-Before you can start building your own connector, you need to understand [Airbyte's data protocol specification](../../architecture/airbyte-specification.md).
+Before building a new connector, review [Airbyte's data protocol specification](../../architecture/airbyte-specification.md).
 
-## Creating a new connector
+## Adding a new connector 
 
-First, make sure you built the project by running
+### Requirements
+To add a new connector you need to: 
+1. Package your connector in an Airbyte Protocol compliant Docker image
+2. Add integration tests for your connector. At a minimum, all connectors must pass Airbyte's standard test suite, but you can also add your own tests.
+3. Add the appropriate Gradle tasks to build the image within the Airbyte monorepo and CI
+
+Each requirement has a subsection below. 
```

Comment:
The headers below should be numbered to match the summaries above.

Reply:
done